Sources/_StringProcessing/RegexDSL/DSL.swift

```diff
@@ -62,6 +62,44 @@ extension CharacterClass: RegexProtocol {
 
 // Note: Quantifiers are currently gyb'd.
 
+/// Specifies how much to attempt to match when using a quantifier.
+public struct QuantificationBehavior {
```

Comment:
Thoughts on this being a "thing" in API vs a set of operations on quantifiable patterns? E.g. there could be a sub-protocol for quantifiables with a `var eagerly: Self` requirement.

(Not arguing one way or another)